sample-spring-boot/pom.xml

```diff
@@ -194,22 +194,22 @@
 				<dependency>
 					<groupId>org.apache.logging.log4j</groupId>
 					<artifactId>log4j-api</artifactId>
-					<version>2.17.2</version>
+					<version>${log4j2.version}</version>
```

Comment:
I was searching for the the definition of log4j2.version in this file but it is specified in the parent pom where it is updated to 2.20.0 with this PR